wallet-new/src/Cardano/Wallet/Kernel/Wallets.hs

```diff
@@ -88,6 +92,8 @@ data UpdateWalletPasswordError =
       -- ^ When trying to update the password inside the keystore, the
       -- previous 'PassPhrase' didn't match or it was deleted, which means
       -- this operation is not valid anymore.
+    | UpdateWalletPasswordUnableForExternalWallet
+      -- External wallets don't have spending password by definition.
```

Comment:
Okay for now, but we'll move forward with defining another type for external wallet which won't have passwords. Then, it won't be necessary to have such error.